test/general/key.js

```diff
@@ -3251,17 +3245,11 @@ module.exports = () => describe('Key', function() {
     const updated = await dest.update(source);
     expect(updated.isPrivate()).to.be.true;
 
-    const { selfCertification: destCertification } = await updated.getPrimaryUser();
-    const { selfCertification: sourceCertification } = await source.getPrimaryUser();
-    destCertification.verified = null;
-    sourceCertification.verified = null;
-    await updated.verifyPrimaryKey().then(async () => expect(destCertification.verified).to.be.true);
-    await source.verifyPrimaryKey().then(async () => expect(sourceCertification.verified).to.be.true);
-
-    destCertification.verified = null;
-    sourceCertification.verified = null;
-    await updated.users[0].verify(updated.keyPacket).then(async () => expect(destCertification.verified).to.be.true);
-    await source.users[0].verify(source.keyPacket).then(async () => expect(sourceCertification.verified).to.be.true);
+    await expect(updated.verifyPrimaryKey()).to.be.fulfilled;
+    await expect(source.verifyPrimaryKey()).to.be.fulfilled;
+
+    await expect(updated.users[0].verify(updated.keyPacket)).to.be.fulfilled;
+    await expect(source.users[0].verify(source.keyPacket)).to.be.fulfilled;
```

Comment:
In these cases I would keep them, since it's otherwise unclear whether we are verifying to test that verifying works, or whether we need verification to test other side-effects IMO

Reply:
Verification should never have side-effects, so IMO it should always be obvious that it's part of the test

Reply:
One day I'll convert you